lib/LambdaHelper.js

```diff
@@ -43,19 +43,25 @@ var _lambdaize = function (userFunc, externals) {
 
         var doneCallback = callbackHandler.getCallback(event, context);
 
-        event.args.push(doneCallback);
+        if (event.args) {
+            event.args.push(doneCallback);
+        }
+
         var externals = /*externals*/null; // The external libraries to install, injected below
         var func = /*user function*/null; // The user function or module to run, injected below
 
         var runUserFunction = function() {
             try {
-            func.apply(this, event.args);
-            }
-            catch(error) {
+                if (event.args) {
+                    func.apply(this, event.args);
+                } else {
```

Comment:
Can you add a comment explaining to others that this must means it's been invoked through an event?